Src/classes/TriggerAccountHandler.cls

```diff
@@ -0,0 +1,44 @@
+public with sharing class TriggerAccountHandler {
```

Comment:
change the Name of the class it should be like AccountTriggerHandler / AccountTriggerHelper